src/commands/lint/configureLintingToolsCommand.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { commands, l10n, window, ExtensionContext } from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import { WorkspaceUtils } from '../../utils/workspaceUtils';
+import { TAB_SPACES } from '../../utils/constants';
+
+const configureLintingToolsCommand =
+    'salesforcedx-vscode-offline-app.configureLintingTools';
+const eslintDependencies = [
+    ['@salesforce/eslint-plugin-lwc-graph-analyzer', '^0.9.0'],
+    ['eslint', '^8.47.0']
+];
+const lwcGraphAnalyzerRecommended: string =
+    'plugin:@salesforce/lwc-graph-analyzer/recommended';
+const eslintRecommended = 'eslint:recommended';
+
+interface PackageJson {
+    devDependencies?: Record<string, string>;
+}
+
+enum MessageType {
+    Error,
+    InformationYesNo,
+    InformationOk
+}
+
+export class ConfigureLintingToolsCommand {
+    static async configure(): Promise<boolean> {
+        try {
+            if (!WorkspaceUtils.lwcFolderExists()) {
+                await this.showMessage('LWC folder does not exist.');
+                return Promise.resolve(false);
+            }
+
+            if (!WorkspaceUtils.packageJsonExists()) {
+                await this.showMessage(
+                    'The project does not contain package.json.'
+                );
+                return Promise.resolve(false);
+            }
+
+            // Ask user to add eslint plugin
+            const result = await this.showMessage(
+                'Do you want to add eslint plugin for LWC data graph anaylsis to your package.json?',
```

Comment:
```suggestion
                'Do you want to add the ESLint plugin for LWC graph analysis to your project? This will give you linting feedback on code patterns that will not support your LWCs working offline, for mobile use cases.',
```